### DISCUSSION TITLE
Cups Server

### DISCUSSION THREAD

**Discussion by terrelliott** [minios-linux/minios-live#32] [Q&A]
I'm trying to use MiniOS as a cups server, but I just can't get it to run. I installed MiniOS Maximum and it doesn't seem to have any print support at all. I got Cups installed and have gotten to the point of trying to install printers. It can find network printers, but fails with "server-error-internal-error". Help!

**Comment by crim50n** [comment]
You should find the instructions for setting up CUPS in Debian 12, they provide quite detailed steps for setting up printers.
